pyuvsim/tests/test_simsetup.py

```diff
@@ -521,7 +612,8 @@ def check_uvdata_keyword_init(case):
 
     elif case == 2:
         # also check that '1' gets converted to [1]
-        # Note -- Currently, pyuvdata's selection does not intersect antenna_nums with bls, but joins them.
+        # Note -- Currently, pyuvdata's selection does not intersect antenna_nums with bls,
+        # but joins them.
```

Comment:
This comment is out of date, it has been fixed to do intersections on pyuvdata.

Reply:
I removed it